data/replaced_tags.txt

```diff
@@ -74,6 +74,7 @@ ice_road=yes : highway=ice_road
 winter_road=yes : highway=ice_road
 highway=turning_loop : highway=mini_roundabout
 highway=platform : highway=pedestrian
+service=emergency_access : access=emergency | u
```

Comment:
the problem is that you can have `service=emergency_access` allowing traffic, especially foot traffic

sometimes also car traffic

Reply:
emergency_access is purely for emergency access when it comes to vehicles: https://wiki.openstreetmap.org/wiki/Tag:service=emergency_access

if specific foot access is tagged, it will still be used for foot routing :)
https://github.com/organicmaps/organicmaps/blob/5fce237ad70dde1ddfd05e59c0d8651c63d546a1/generator/road_access_generator.cpp#L113-L120

Reply:
I would assume that `highway=service service=emergency_access` allows pedestrians.

`highway=service service=emergency_access access=emergency` does not allow pedestrians.

Reply:
Looks like OSRM, Valhalla & GraphHopper handle it as if only vehicles are blocked, allowing bikes and foot, so you're right. Will fix it 👍  (https://www.openstreetmap.org/directions?engine=graphhopper_foot&route=51.41009%2C0.49669%3B51.40999%2C0.49833#map=19/51.40987/0.49751)

Reply:
Also, how it works if access is set already? Is it overriding it?

Reply:
yes, `| u` will override the existing tag